data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -211,13 +211,17 @@ then
 	# 1. read the controller config rendered by MachineConfigOperator
 	# 2. read the default MachineConfigPools rendered by MachineConfigOperator
 	# 3. read any additional MachineConfigs that are needed for the default MachineConfigPools.
-	mkdir --parents /etc/mcc/bootstrap /etc/mcs/bootstrap /etc/kubernetes/manifests
+	mkdir --parents /etc/mcc/bootstrap /etc/mcs/bootstrap /etc/kubernetes/manifests /etc/kubernetes/static-pod-resources
 	cp mco-bootstrap/bootstrap/manifests/* /etc/mcc/bootstrap/
 	cp openshift/* /etc/mcc/bootstrap/
 	# 4. read ImageContentSourcePolicy objects generated by the installer
 	cp manifests/* /etc/mcc/bootstrap/
 	cp auth/kubeconfig-kubelet /etc/mcs/kubeconfig
 	cp mco-bootstrap/bootstrap/machineconfigoperator-bootstrap-pod.yaml /etc/kubernetes/manifests/
+	if [ -d mco-bootstrap/openstack/manifests ]; then
+	    cp mco-bootstrap/openstack/manifests/* /etc/kubernetes/manifests/
+	    cp -r mco-bootstrap/openstack/static-pod-resources/* /etc/kubernetes/static-pod-resources/
```

Comment:
nit: I think this script uses full names for flags.. `--recursive`

Reply:
We just used what baremetal did, but I agree being explicit is better. Fixed.